sdk/src/main/java/io/dapr/utils/Constants.java

```diff
@@ -17,13 +17,20 @@
     /**
      * Dapr's default hostname.
      */
-    public static final String DEFAULT_HOSTNAME = "localhost";
+    public static final String DEFAULT_HOSTNAME = "http://localhost";
```

Comment:
hostname is "localhost", consumers of that constant expect it to be an actual hostname (without "http://").
"http://localhost/" should be called DEFAULT_HTTP_BASE_URL or something.